src/middlewared/middlewared/plugins/crypto_/certificates.py

```diff
@@ -110,25 +110,28 @@ async def cert_services_validation(self, id, schema_name, raise_verrors=True):
                     f'{cert["name"]}\'s private key size is less then {valid_key_size[cert["key_type"]]} bits'
                 )
 
-            if cert['until'] and datetime.datetime.strptime(
-                cert['until'], '%a %b  %d %H:%M:%S %Y'
-            ) < datetime.datetime.now():
-                verrors.add(
-                    schema_name,
-                    f'{cert["name"]!r} has expired (it was valid until {cert["until"]!r})'
-                )
-
-            if cert['digest_algorithm'] in ['MD5', 'SHA1']:
-                verrors.add(
-                    schema_name,
-                    'Please use a certificate whose digest algorithm has at least 112 security bits'
-                )
-
-            if cert['revoked']:
-                verrors.add(
-                    schema_name,
-                    'This certificate is revoked'
-                )
+            if not cert['cert_type_CSR']:
```

Comment:
I think it's better to just not validate rest of the stuff at all if the cert is not okay - like why should we care what the key size if the selection in question is a CSR which is wrong ?